apps/sharedtest/test/unit/crop_resize_rotate_test.js

```diff
@@ -4,13 +4,95 @@
 /* global parseJPEGMetadata */
 /* global getImageSize */
 /* global Downsample */
-/* global cropResizeRotate */
+/* global cropResizeRotate, JPEG */
 
+require('/shared/test/unit/mocks/mock_lazy_loader.js');
 require('/shared/js/blobview.js');
 require('/shared/js/media/jpeg_metadata_parser.js');
 require('/shared/js/media/image_size.js');
 require('/shared/js/media/downsample.js');
 require('/shared/js/media/crop_resize_rotate.js');
+require('/shared/js/media/jpeg-exif.js');
+
+function fetchBuffer(url) {
+  return new Promise(function(resolve, reject) {
+    var xhr = new XMLHttpRequest();
+    xhr.open('GET', url);
+    xhr.onload = function() {
+      if (xhr.status !== 200) {
+        reject(new Error('Failed with status: ' + xhr.status));
+      } else {
+        resolve(this.response);
+      }
+    };
+    xhr.onerror = xhr.ontimeout = function() {
+      reject(new Error('Failed' + xhr.status));
+    };
+    xhr.responseType = 'arraybuffer';
```

Comment:
can you just use `responseType = 'blob'` here and skip the fetchBlob() function below?